### PR TITLE
[COST-3896] Add product/physicalCores and identity/TimeInterval to the AWS Line Items

### PR DESCRIPTION
**Pull request by djnakabaale**
## Jira Ticket

[COST-3896](https://issues.redhat.com/browse/COST-3896)

## Description

This change will add relevant AWS SWATCH columns to `aws_line_items`

## Testing

1. Checkout Branch
2. Restart Koku
3. Drop the `aws_line_items` trino table with `python scripts/cji_scripts/migrate_trino.py`
4. Run the following commands to load AWS test data
    ```
    make create-test-customer
    make load-test-customer-data test_source=aws
    ```
5. launch trino shell and check for the new columns.
    ```
     trino --server 127.0.0.1:8080 --catalog hive --schema=org1234567 --user admin --debug

     SHOW COLUMNS FROM aws_line_items LIKE  '%interval';
    ```
    1. You should see the `identity_timeinterval` column.
    2. If you have the changes from the nise pr linked in the notes section, you should see also the `product_physicalcores` column when you run the command below otherwise zero columns will be returned.
    ```
     SHOW COLUMNS FROM aws_line_items LIKE  '%cores';
    ```

## Notes

...
Requires https://github.com/project-koku/nise/pull/453 for appropriate test data

### CODE REVIEW

File: koku/masu/util/aws/aws_post_processor.py [hunk 1]
```diff
@@ -86,6 +86,8 @@ class AWSPostProcessor:
         "reservation/UnitsPerReservation",
         "reservation/StartTime",
         "reservation/EndTime",
+        "product/physicalCores",
+        "identity/TimeInterval",
```
[the remark this answers is not in the document]
Comment (lcouzens) [reply koku/masu/util/aws/aws_post_processor.py:90]: Also is `product/physicalCore` even a real column? :grimacing:

**Review by lcouzens — CHANGES_REQUESTED** [review]
We need to check that `product/physicalCores` thing since thats not a real column in any of the real AWS data I've looked at. We have `product/vcpus` maybe we want that? Either way we dont want to add this column to the required ingress columns since thats going to likely break customer filtered data posts today.

File: koku/masu/util/aws/aws_post_processor.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (djnakabaale) [reply koku/masu/util/aws/aws_post_processor.py]: 😃 `product/physicalCores` is a real [column in the AWS Cost and Usage Reports](https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html#product-details-P)

**Comment by djnakabaale** [comment]
> We need to check that `product/physicalCores` thing since thats not a real column in any of the real AWS data I've looked at. We have `product/vcpus` maybe we want that? Either way we dont want to add this column to the required ingress columns since thats going to likely break customer filtered data posts today.

This is addressed in these comments

https://github.com/project-koku/koku/pull/4435#discussion_r1228210975 and https://github.com/project-koku/koku/pull/4435#discussion_r1228210463

**Review by lcouzens — APPROVED** [review]

**Pull request merged**